docs/api/native-image.md

```diff
@@ -137,15 +137,17 @@ let image = nativeImage.createFromPath('/Users/somebody/images/icon.png')
 console.log(image)
 ```
 
-### `nativeImage.createFromBuffer(buffer[, scaleFactor])`
+### `nativeImage.createFromBuffer(buffer[width, height, scaleFactor])`
 
 * `buffer` [Buffer][buffer]
+* `width` Integer (optional)
```

Comment:
Hmmm, the length check does appear to make it backwards compatible.  Still not too comfortable with the args ordering as without looking at the implementation in a lot of depth it **looks** breaking 😆 

Fine with it though if it isn't actually breaking